Cargo.toml

```diff
@@ -44,7 +44,7 @@ version = "0.1.3"
 default_features = false
 features = ["display_link"]
 
-[target.'cfg(any(target_os = "ios", target_os = "windows"))'.dependencies]
+[target.'cfg(any(target_os = "ios", target_os = "windows", target_os = "linux"))'.dependencies]
```

Comment:
Quick note - this will cause the build to fail for BSD and Dragonfly systems, since those systems use our Linux backend. Replace this with `target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"`, as seen later down in this file.

Reply:
Ahh!  I didn't even think of that.  Should be resolved now.